### ISSUE TITLE
I'm having trouble loading the trained model

### ISSUE THREAD

**Issue by MATLABmdl** [ultralytics/ultralytics#2824]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

Hello

I'm using yolo8 to training pose model
yolo8-pose

requirements: YOLOv8 requirement "ultralytics.nn.modules.conv" not found, attempting AutoUpdate...
requirements:  Command 'pip install --no-cache "ultralytics.nn.modules.conv"  ' returned non-zero exit status 1.

I'm train a model in googlecolab and I'cant load it on my own system

train code
model.train(data='project.yaml', epochs=300, imgsz=640, augment=True, int8=True)
system : '8.0.93'
google : 8.0.108

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by MATLABmdl** [comment]
I can send the model, can that help?

**Comment by glenn-jocher** [comment]
Hello @MATLABmdl, 

Thank you for reaching out. 

We are happy to help you with your question about loading a trained YOLOv8 model on your win-64 system. 

Regarding your question, we are unable to examine the accuracy of the provided model and ask that you provide further details regarding the error message you receive when attempting to load it. 

Additionally, we request that you observe caution when downloading models from third-party websites as they are not verified by the YOLOv8 team. 

We strongly encourage that you only download models from our official model zoo or train your own models. 

If you have any further questions or concerns, please do not hesitate to ask.

Best regards.

**Comment by glenn-jocher** [comment]
@MATLABmdl if your model is not loading properly on the latest version of ultralytics, you may need to train a new model, as we recently made significant updates to you modules section. They are intended to be backwards compatible, so everything should work correctly, but if not then I would retrain a new model with the latest package, and then everything will work correctly.
